e2e/pages/caseView.page.js

```diff
@@ -71,12 +71,13 @@ module.exports = {
   },
 
   checkTaskStatus(task, status) {
-    I.waitForElement(locate(`//p/a[text()="${task}"]`), 10);
+    const taskElement = `//p/a[text()="${task}"]`;
+    I.waitForElement(locate(taskElement), 10);
+    I.scrollIntoView(taskElement);
     if(status) {
-      I.seeElement(locate(`//p/a[text()="${task}"]/../img`).withAttr({title: status}));
+      I.waitForElement(locate(`${taskElement}/../img`).withAttr({title: status}), 10);
```

Comment:
both have the `/../img`, could just be a new variable if you want

Reply:
yeah I was thinking the same, but it turns out that the `/../img/` part is related to the status icon, so for the `I.dontSeeElement()` step which is checking that the _status_ isn't displayed we still want to know that the _task_ is present - which happens on line 75.
otherwise the check could pass on a totally different page or if the page hadn't finished loading